packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -794,6 +821,25 @@ class CupertinoSliverNavigationBar extends StatefulWidget {
   /// {@macro flutter.cupertino.CupertinoNavigationBar.heroTag}
   final Object heroTag;
 
+  /// A widget to place at the bottom of the large title or static navigation
+  /// bar if there is no large title.
+  ///
+  /// Only widgets that implement [PreferredSizeWidget] can be used at the
+  /// bottom of a navigation bar.
+  ///
+  /// See also:
+  ///
+  ///  * [PreferredSize], which can be used to give an arbitrary widget a preferred size.
+  final PreferredSizeWidget? drawer;
```

Comment:
Wondering if `drawer` is a less intuitive name than `bottom`. In cases like this, should we prefer the HIG naming conventions over our own APIs (or what we think is best)?

Reply:
`bottom` is probably better so we match our own APIs. The word drawer should be used in the docs though, to direct people that are used to HIG naming.